rd_ui/app/views/dashboard.html

```diff
@@ -58,7 +58,7 @@ <h3 class="panel-title">
                 </div>
             </div>
 
-            <div class="panel panel-default rd-widget-textbox" ng-if="type=='textbox'" ng-mouseenter="showControls = true" ng-mouseleave="showControls = false">
+            <div class="panel panel-default rd-widget-textbox" ng-style="widget.disableWidget" ng-if="type=='textbox'" ng-mouseenter="showControls = true" ng-mouseleave="showControls = false">
```

Comment:
Are you sure it was ng-hide and not ng-if?
ng-hide just applies `display:none` to the element, while ng-if doesn't render it. In theory, there shouldn't be difference between what you did and ng-hide, except that in your case it might take height on the page but still will be invisible.